v1/OpsGenieConfig.dhall

```diff
@@ -0,0 +1,58 @@
+let imports = ../imports.dhall
+
+let Kubernetes = imports.Kubernetes
+
+let BasicAuth = ./BasicAuth.dhall
+
+let HTTPConfig = ./HTTPConfig.dhall
+
+let OpsGenieConfigDetail =
+      { Type = { key : Text, value : Text }, default = {=} }
+
+let OpsGenieConfigResponder =
+      { Type =
+          { id : Optional Text
+          , name : Optional Text
+          , username : Optional Text
+          , type : Optional Text
+          }
+      , default =
+        { id = None Text
+        , name = None Text
+        , username = None Text
+        , type = None Text
+        }
+      }
```

Comment:
Just to keep the convention in the repo seperate it to different file OpsGenieConfigResponder.dhall